pipeline.yml

```diff
@@ -1,43 +1,135 @@
 ---
+groups:
+- name: all
+  jobs:
+  - unit-test-uaa-guard-service
+  - push-uaa-guard-service-staging
+  - push-uaa-guard-service-production
+- name: tests
+  jobs:
+  - unit-test-uaa-guard-service
+  # TODO: Once assertion tests are created
+  #- assertion-tests-uaa-guard-service
```

Comment:
@rememberlenny here's a good example. They shouldn't be too difficult to add. 

https://github.com/18F/cf-cdn-service-broker/blob/93cbea8487230b66a8258ff0572d26bcdcacbede/ci/acceptance-tests.sh